.dockerignore

```diff
@@ -0,0 +1,4 @@
+.*
```

Comment:
I'm slightly confused by this Dockerignore contents. Wouldn't it be simpler to ignore everything and un-ignore the required files? I've tried to do something like this in the [C# test runner](https://github.com/exercism/csharp-test-runner/blob/master/.dockerignore).

Reply:
Agreed. Thanks.